activejdbc/src/test/java/org/javalite/activejdbc/ModelTest.java

```diff
@@ -482,6 +482,19 @@ public void shouldGenerateCorrectInsertSQL() {
         insertSQL = s.toInsert(new SimpleFormatter(java.sql.Date.class, "to_date('", "')"));
         the(insertSQL).shouldBeEqual("INSERT INTO students (dob, first_name, id, last_name) VALUES (to_date('1965-12-01'), 'Jim', 1, 'Cary')");
     }
+    
+    @Test
+    public void shouldGenerateCorrectUpdateSQL(){
+        Student s = new Student();
+        s.set("first_name", "Jim");
+        s.set("last_name", "Cary");
+        s.set("dob", new java.sql.Date(getDate(1965, 12, 1).getTime()));
+        s.set("id", 1);
+        String updateSQL = s.toUpdate();
+        System.out.println(updateSQL);
+
+        the(updateSQL).shouldBeEqual("UPDATE students SET dob = DATE '1965-12-01', first_name = 'Jim', id = 1, last_name = 'Cary' WHERE id = 1");
```

Comment:
Don't test for the String content (that will change for other DBMSs), but instead call `Base.exec(updateSQL)` and check if it updated correctly, like this:

1. Do `deleteAndPopulateTable("students");`
2. Do `Student s = Student.findById(1);` and change some of the `s` attributes' values.
3. Do `Base.exec(updateSQL);`
4. Do `s = Student.findById(1);` again and check if it has changed values.